Code/GraphMol/NewCIPLabelling/BaseMol.hpp

```diff
@@ -0,0 +1,95 @@
+//
+//
+//  Copyright (C) 2020 Greg Landrum and T5 Informatics GmbH
```

Comment:
I know we talked about it before, but I will point out again just to be clear: I don't *need* to be the copyright holder on these files.

Reply:
fine to leave things this way, but it's not necessary

Reply:
Neither do I :)

Reply:
I think we should probably be (C) Schrodinger, LLC, since they're paying for the work.

Reply:
Ok.